### PR TITLE
44 styling edit profile profile

### LINKED ISSUE #44: Edit Profile & Profile styling

### PR DESCRIPTION
**Pull request by HachemBouhadede**
styling, responsiveness, translation, fixed some minor bugs for profile page
styling, responsiveness, translation, fixed some minor bugs for edit profile page
language switch changes to work around the i18 and dynamic routes (related to SSG and Link/Next requirement for dynamic pages)
some path changes

# Related Issue

- Resolve #44

### CODE REVIEW

**Comment by HachemBouhadede** [comment]
on hold, issues with the deployment....

**Comment by HachemBouhadede** [comment]
@202306-NEA-DZ-FEW/team-nine if anyone can step up for this feel free !

- tried to change next-i18 config (the current version is changed)
- change the language path in the navbar
- tried to switch from statisProps to serverSideProps, it eended up having the website not launching

File: src/lib/firebase/users.js
Comment (Khalil-NOUI) [inline src/lib/firebase/users.js:1]: what's this for ?

**Review by Khalil-NOUI — COMMENTED** [review]
well done, Some code files are bit vague but nerveless good job

**Comment by HachemBouhadede** [comment]
it'sssssssssssssssss working :')))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))))

**Pull request merged**